src/estimators/stratified_sketch.py

```diff
@@ -0,0 +1,242 @@
+# Copyright 2020 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import collections
+import copy
+import functools
+import sys
+import numpy as np
+from wfa_cardinality_estimation_evaluation_framework.estimators.base import SketchBase
+from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import ExactMultiSet
+
+ONE_PLUS = '1+'
+
+class StratifiedSketch(SketchBase):
```

Comment:
I am not sure if StratifiedSketch should inherit from the SketchBase. Seems like it is pretty different from the sketches we usually refer to, eg, bloom filters. I would like to hear more opinions from other reviewers.

Reply:
@matthewclegg what do you think?

Reply:
I agree with Xichen.